ACPAcquisition.podspec

```diff
@@ -1,29 +1,26 @@
 Pod::Spec.new do |s|
   s.name         = "ACPAcquisition"
-  s.version      = "1.1.0"
-  s.summary      = "Acquisition framework for Adobe Experience Platform SDK. Written and Supported by Adobe."
+  s.version      = "2.0.0"
+  s.summary      = "Acquisition library for Adobe Experience Cloud SDK. Written and maintained by Adobe."
   s.description  = <<-DESC
-                   The Acquisition framework provides APIs that allow use of acquisition and Adobe link features in the Adobe Experience Platform SDK.
+                   The Acquisition library provides APIs that allow use of the Acquisition product in the V5 Adobe Experience Cloud SDK.
                    DESC
 
   s.homepage     = "https://github.com/Adobe-Marketing-Cloud/acp-sdks/releases"
 
   s.license      = {:type => "Commercial", :file => "LICENSE.md"}
   s.author       = "Adobe Experience Platform SDK Team"
-  s.source       = { :git => 'https://github.com/Adobe-Marketing-Cloud/acp-sdks.git', :tag => "v1.1.0-ACPAcquisition" }
+  s.source       = { :git => 'https://github.com/Adobe-Marketing-Cloud/acp-sdks.git', :tag => "v2.0.0-ACPAcquisition" }
```

Comment:
Can we use this format `"v#{s.version}-#{s.name}"` so next time we don't need to update the version in two places?